pkg/workflows/wasm/host/wasm_test.go

```diff
@@ -903,6 +903,48 @@ func TestModule_Sandbox_Memory(t *testing.T) {
 	assert.ErrorContains(t, err, "exit status 2")
 }
 
+func TestModule_CompressedBinarySize(t *testing.T) {
+	t.Parallel()
+
+	t.Run("compressed binary size is smaller than the default 10mb limit", func(t *testing.T) {
+		binary := createTestBinary(successBinaryCmd, successBinaryLocation, false, t)
+
+		_, err := NewModule(&ModuleConfig{IsUncompressed: false, Logger: logger.Test(t)}, binary)
+		require.NoError(t, err)
+	})
+
+	t.Run("compressed binary size is bigger than the default 10mb limit", func(t *testing.T) {
+		// 11mb binary
+		binary := make([]byte, 11*1024*1024)
```

Comment:
```suggestion
		binary := make([]byte, defaultBinarySize + 1)
```
_nit_
maybe simpler to keep the test in sync if this default ever changes